### PR TITLE
Release tool fixes

### PR DESCRIPTION
**Pull request by kacf**

### CODE REVIEW

**Comment by kacf** [comment]
It's easier to view this with whitespace off.

**Comment by kacf** [comment]
@mender-test-bot start pipeline

File: extra/test_release_tool.py [hunk 1]
```diff
@@ -194,38 +139,70 @@ def test_version_of(capsys):
...
+    # At the time of writing, workflows git repository has two docker images,
+    # where one has the same name as the repository (ambiguous). Make sure it
+    # still works to query this when specifically giving "docker" as the version
+    # type.
     run_main_assert_result(
-        capsys, ["--version-of", "gui", "--version-type", "git"], "1.2.3-git"
+        capsys,
+        ["--version-of", "workflows", "--version-type", "docker"],
+        "mender-master",
     )
```
Comment (lluiscampos) [inline extra/test_release_tool.py:206]: For completeness, I would add a test for the git variant:
```
    run_main_assert_result(
        capsys,
        ["--version-of", "workflows", "--version-type", "git"],
        "master",
    )
```

**Review by lluiscampos — APPROVED** [review]
I agree with skipping this test on non-master runs. Good clean-up!

**Review by oleorhagen — APPROVED** [review]

File: extra/test_release_tool.py (hunk 1 above)
Comment (kacf) [reply extra/test_release_tool.py:206]: Good point, added.

**Comment by kacf** [comment]
@mender-test-bot start pipeline

@lluiscampos, @oleorhagen: Updated, and added the fix for the mender-client-docker-addons query.

**Comment by kacf** [comment]
@mender-test-bot start pipeline

**Comment by kacf** [comment]
@mender-test-bot start pipeline

File: extra/release_tool.py [hunk 2]
```diff
@@ -583,17 +583,31 @@ def version_of(
...
+                % (
+                    component.name,
+                    component.type,
+                    ", ".join([c.name for c in docker_components]),
+                )
             )
         image_name = docker_components[0].docker_image()
 
-    if git_components[0].git() == "integration":
+    if git_version and git_components[0].git() == "integration":
```
Comment (lluiscampos) [inline extra/release_tool.py:610]: :smile:

**Review by lluiscampos — APPROVED** [review]

**Pull request merged**